tutorial/tutorial-gabc-01.html

```diff
@@ -130,14 +161,15 @@
 	</pre>
 </div>
 
+<p>Now, if you try to typeset this document, you're going to get an error indicating there's a problem in <code>kyrie.gabc</code>.  We're going to fix this error later, so for now (and for the next few steps) simply instruct <span class="tex">T<span class="epsilon">e</span>X</span> to ignore the error.  After it finishes the typesetting process (without producing a pdf), typeset the document again and you won't get the error, but will get the pdf file.</p>
```

Comment:
Maybe there could be a hint on how to instruct TeX to ignore the error? maybe `typing x`?